tar/read.c

```diff
@@ -67,6 +67,8 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/read.c,v 1.40 2008/08/21 06:41:14 kientzle E
 
 #include "bsdtar.h"
 #include "err.h"
+#include "archive_platform.h"
+#include "archive_entry.h"
```

Comment:
archive_entry.h is already included via bsdtar_platform.h

Including it again should not be necessary.